src/enclave-agent/src/services/images.rs

```diff
@@ -111,6 +99,29 @@ impl ImageService {
 
         Ok(image.to_owned())
     }
+
+    fn parse_container_id(&self, req: &image::PullImageRequest) -> Result<String> {
+        let mut cid = req.get_container_id().to_string() ;
+        
+        if cid.is_empty() {
+            let image = req.get_image();
+            let v: Vec<&str> = image.rsplit('/').collect();
+            if !v[0].is_empty() {
+                cid = v[0].to_string()
+            } else {
+                return Err(anyhow!("Invalid image name: {:?}", image));  
+            }
+        }
+    
+        // FIXME: verify_id is borrowed from kata container CCv0
+        // which use umoci to unpack image. And ':' have special 
+        // meaning for umoci during upack. We need new functions 
+        // to generate cids and verify them in the future.
```

Comment:
`verify_id` is used from `kata-sys-util` and addressed part of #15 so what this borrowing mean here?

Reply:
The "borrow" means that the function is defined in an outside crate `kata-sys-util` and we can not modify it directly. But as the #15 said, it is an acceptable design, I will reorganize the comment.